src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -47,7 +46,7 @@ case class Instance(
     *
     * Note: A provisioned instance is considered active.
     */
-  val isScheduled: Boolean = state.goal == Goal.Running && !isActive && !state.condition.isLost && state.condition != UnreachableInactive
+  val isScheduled: Boolean = state.goal == Goal.Running && (state.condition.isTerminal || state.condition == Condition.Scheduled)
```

Comment:
What is the reason for this change?

Reply:
@meichstedt ?

Reply:
IIRC mainly because it's easier to understand – if the instance is in a terminal condition or explicitly scheduled, we consider it as to be scheduled. Lost/Unreachable/UnreachableInactive are not terminal states, so these will never be scheduled.